content/applications/productivity/whatsapp.rst

```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+overview the process for creating templates in Odoo.
+
+.. important::
+   WhatsApp has an approval process that must be completed before the template can be used.
+
+To access WhatsApp templates go to the :menuselection:`Whatsapp app --> Templates --> New`.
+
+.. _whatsapp/templates:
+
+Creating templates
+------------------
+
+Templates are prefabricated layouts that allow users to professional looking messages to customers.
+These templates can contain dynamic data that will populate in the end message using variables that
+are set in the template configuration.
+
+To create a WhatsApp template, click on :guilabel:`New`, and set the :guilabel:`Name`. Then select
+the :guilabel:`Language` for the template.
+
+Under the :guilabel:`Account` select the *WhatsApp business account* in Odoo that this template
+should link to. Next, under the :guilabel:`Applies to` field select the *model* the server action
+for this template, will *apply to*.
+
+.. tip::
+   These models can also be accessed in :ref:`developer mode <developer-mode>`. On a contact form
+   (or other relevant form in Odoo), navigate to the model that will be referenced, and hover over
+   any field name. A box of backend information will reveal itself with the specific Odoo
+   :guilabel:`Model` name in the backend. Search (using front-end name) for this model in the
+   :guilabel:`Applies to` drop-down menu in the WhatsApp template.
+
+Often when changing the model or :guilabel:`Applies to` field the :guilabel:`Phone Field` made
+produce an error. The :guilabel:`Phone field `should always result in `Phone` or `Mobile`. To search
+the available fields, type in the front-end name (on user-interface) of the field in the search box.
+This will find a result from all of the available fields for the model (:guilabel:`Applies to`) that
+the template is created for. There may be multiple levels that need to be configured.
+
+Change the :guilabel:`Category` to fit either a :guilabel:`Marketing`, :guilabel:`Utility`, or
+:guilabel:`Authentication` category. Specifying an incorrect category can cause a flag from Meta In
+the approval process.
+
+Add and :guilabel:`User` that are allowed to use this template. On the right hand column of the top
+configurations a :guilabel:`Header type` can be configured along with a :guilabel:`Header message`.
+
+The available :guilabel:`Header types` are as follows:
+
+- Text
+- Image
+- Video
+- Document
+- Location
+
+Navigate to the :guilabel:`Body` tab to configure the main message of the template.
+
+When all the necessary changes are made to the template, then click on the :guilabel:`Submit for
+approval` button in the upper left hand corner. The status of the template will change to
+:guilabel:`Pending`. The status will remain in :guilabel:`Pending` until a decision has been made by
+Meta. Once the user receives an email confirming that the template has been approved the templates
+will need to be synced from the Odoo database.
```

Comment:
```suggestion
must be synced from the Odoo database.
```

i feel "must" is strong, and it won't work unless they do it, so it seemed fitting.

Reply:
It is a must do